### ISSUE TITLE
2025 Accessibility Report: Search results touch targets

### ISSUE THREAD

**Issue by bertrama** [mlibrary/call-number-mapping#13]
On the Search LC/Dewey page:

The touch targets for the result links are too small. This is currently not a violation in WCAG 2.1 but is in 2.2 AA

All touch targets must be 24px large, or leave sufficient space. Make the links, table rows larger
